cobc/codegen.c

```diff
@@ -1110,6 +1114,8 @@ output_base (struct cb_field *f, const cob_u32_t no_output)
 		output ("%s%d", CB_PREFIX_BASE, f01->id);
 	}
 
+	/* CHECKME: May varying addresses occur only with sliging ODO?  If yes
```

Comment:
What is "sliging"? Maybe "sliding"?
In general every BASED item and item in LINKAGE may have a variable address, too - but I'm not sure that this part is about that.